tensorflow/contrib/layers/python/layers/normalization.py

```diff
@@ -93,6 +94,9 @@ def instance_norm(inputs,
   inputs = ops.convert_to_tensor(inputs)
   inputs_shape = inputs.shape
   inputs_rank = inputs.shape.ndims
+  # Cast `inputs` array to float64 if machine is big endian
```

Comment:
This code comment could be better. Reading the code, I see what this does, but I don't know why. Could you expand on that?

Reply:
Updated comments as below:

  # For big endian, precision difference in last decimal values getting in 
  # float32 Vs float64 data type is causing normalization_test failure. 
  # The cast to float64 will calculate mean and variance correctly while 
  # normalization of `inputs` tensor.

Reply:
@martinwicke Gentle reminder